apps/src/constants.js

```diff
@@ -189,4 +189,4 @@ export const NOTIFICATION_ALERT_TYPE = 'notification';
 
 export const START_BLOCKS = 'start_blocks';
 
-export const DEPRECATED_LABS = ['calc', 'eval'];
+export const DEPRECATED_PROJECT_TYPES = ['algebra_game', 'calc', 'eval'];
```

Comment:
I renamed this because 'algebra_game' is technically Play Lab, and so these types aren't really full "labs".